hack/api_transformer/main.go

```diff
@@ -328,15 +328,19 @@ func processDocs(file *ast.File) {
 		case *ast.GenDecl:
 			convertHideFromDocs(x.Doc)
 			convertTabsToHeadings(x.Doc)
+			cleanComments(x.Doc)
```

Comment:
We can now introduce a new function that calls all these three functions, so we don't have to repeat calling all three for every `case` in this `switch`.